ui/src/widgets/ui-form/UIForm.vue

```diff
@@ -153,6 +153,9 @@ export default {
                 return []
             }
         },
+        formattedLabel (row) {
+            return row.required ? `${row.label} (*)` : row.label
```

Comment:
```suggestion
            return row.required ? `* ${row.label}` : row.label
```

I think it reads better without the `()` and with the `*` up front of the text